utilities/backupable/backupable_db_test.cc

```diff
@@ -766,6 +856,139 @@ class BackupableDBTestWithParam : public BackupableDBTest,
   }
 };
 
+TEST_F(BackupableDBTest, DbAndBackupSameCustomChecksum) {
```

Comment:
Did you add the testing case to cover the backward compatible?

Reply:
Which backward compatible case you meant? If you meant no custom checksum factory for the backup, I think the existing tests (excluding those added by this PR) already cover it :P